File: java/3254. Find the Power of K-Size Subarrays I.java
class Solution {
    public int[] resultsArray(int[] nums, int k) {
        int[] answer = new int[nums.length-k+1];
        LOOP: for (int i = 0; i < nums.length-k+1; i++) {
            for (int j = i+1; j < i+k; j++) {
                if (nums[j] != nums[j-1]+1) {
                    answer[i] = -1;
                    continue LOOP;
                }
            }
            answer[i] = nums[i+k-1];
        }
        return answer;
    }
}

File: go/3254. Find the Power of K-Size Subarrays I.go
func resultsArray(nums []int, k int) []int {
    result := make([]int, len(nums)-k+1)
    LOOP: for i := 0; i < len(nums)-k+1; i++ {
        for j := i+1; j < i+k; j++ {
            if nums[j] != nums[j-1]+1 {
                result[i] = -1
                continue LOOP
            }
        }
        result[i] = nums[i+k-1]
    }
    return result
}

